### ISSUE TITLE
TODO before deployment of Bonding V2 - withdrawal of pending UBQ

### ISSUE THREAD

**Issue by zgorizzo69** [ubiquity/ubiquity-dollar#155]
* UBQ rewards from BondingV1 can be withdrawn  when UBQ Minter role is removed and there is no more lp sitting on the bonding contract  (as explained to @zapaz )

```javascript
await resetFork(12903140);
// setter le bondingcontractAdr  
await manager.setBondingContractAddress(bondingV2.address);
// remove role UBQ_MINTER et BURNER du masterchef et bonding contract
 await manager.connect(admin).revokeRole(UBQ_MINTER_ROLE, masterChef.address);
  await manager.connect(admin).revokeRole(UBQ_MINTER_ROLE, bonding.address);
  await manager.connect(admin).revokeRole(UBQ_BURNER_ROLE, bonding.address);
// send the LP token from bonding V1 to V2 to prepare the migration
  await bonding.sendDust(
    bondingV2.address,
    metaPool.address,
    bondingMinBalance.add(bondingMaxBalance)
  );
assert masterchef.getRewards() still working 
```

**Comment by zgorizzo69** [comment]
merged with ubiquity/uad-contracts#34 alongside pausable tests

**Comment by 0x4007** [comment]
https://etherscan.io/tx/0x1fc0e8c52d37ba5ab26619697d34536c93c3192ed42418c79d2cf6a5aa6e58fb
